packages/menu/src/Menu/Menu.types.ts

```diff
@@ -12,7 +12,9 @@ export type SubMenuType = ReactElement<
   InferredPolymorphicPropsWithRef<PolymorphicAs, SubMenuProps>
 >;
 
-export interface MenuProps extends Omit<PopoverProps, 'active'> {
+export interface MenuProps
+  extends Omit<PopoverProps, 'active'>,
+    Omit<ComponentPropsWithoutRef<'div'>, 'onClick'> {
```

Comment:
If I'm reading this correctly, this omits the `onClick` from `div`, but `PopoverProps` will still include `onClick`?

Reply:
Yeah we omit the `onClick` from div props since it collides with the one defined in `PopoerProps`